src/Features/CSharp/Portable/CSharpFeaturesResources.resx

```diff
@@ -461,4 +461,22 @@
   <data name="NamespaceName" xml:space="preserve">
     <value>&lt;namespace name&gt;</value>
   </data>
+  <data name="AutoselectDisabledDueToClassDeclaration" xml:space="preserve">
```

Comment:
I actually had that initially, but then I realised I was just being lazy :-) Unless there's localisation overhead you want to avoid, I'd like to leave it as is.